block.go

```diff
@@ -1404,7 +1436,8 @@ func (p *parser) paragraph(data []byte) int {
 			// did this blank line followed by a definition list item?
 			if p.flags&DefinitionLists != 0 {
 				if i < len(data)-1 && data[i+1] == ':' {
-					return p.list(data[prev:], ListTypeDefinition)
+					ret := p.list(data[prev:], ListTypeDefinition)
+					return ret
```

Comment:
Ohh, that's an oversight. I must've done it to debug-print and forgot to change back.